libs/domain/picking/src/components/picking-list-item/picking-list-item.styles.ts

```diff
@@ -0,0 +1,64 @@
+import { css } from 'lit';
+
+export const styles = css`
+  oryx-card {
+    --oryx-card-header-padding: 16px 10px 12px;
+    --oryx-card-body-padding: 9px 10px 12px;
+    --oryx-card-footer-padding: 0 10px 12px;
```

Comment:
Assume should be fixed on design side

Reply:
Started a conversation with the design team here https://spryker.slack.com/archives/C03NTPT5ZTQ/p1678827556664729